packages/workbox-webpack-plugin/src/lib/get-webpack-manifest-entries.js

```diff
@@ -0,0 +1,76 @@
+const url = require('url');
```

Comment:
To prevent developers from precaching huge files (i.e. videos / audio files) we have a size limit on the files we'll precache. Is that possible with WebPack?

Reply:
Perhaps something like https://github.com/ai/size-limit/blob/master/index.js#L135 would do the trick?

Reply:
TBH I think it was static and something we internally enforced. @jeffposnick do you know if this was something we otherwise made configurable?

Reply:
It's configurable via the `maximumFileSizeToCacheInBytes` option.